### PR TITLE
404 fix /docs/

### PR DESCRIPTION
**Pull request by felixwidjaja**
#42 
Using ohdear.app it found several dead links. Mostly due to recent URL change.

### CODE REVIEW

File: docs/services/osie.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ OSIE can built, run, and tested outside of the Tinkerbell stack. Take a look at
...
 
-OSIE gets compiled to a initial ramdisk and a kernel. The initial ramdisk contains Docker and it starts the [`tink-worker`](/docs/services/tink) as a docker container. The `tink-worker` is the application that gets and manage workflows. Usually one of the first workflows it generates is the one that installs an operating system.
+OSIE gets compiled to a initial ramdisk and a kernel. The initial ramdisk contains Docker and it starts the [`tink-worker`](/services/tink) as a docker container. The `tink-worker` is the application that gets and manage workflows. Usually one of the first workflows it generates is the one that installs an operating system.
 
 Currently, the OSIE of today is a bit "too fat"! We are in the process of moving a lot of the customization out of OSIE and into workflows.
 
 ## When You See OSIE
 
-If you follow the [Vagrant Setup](/docs/setup/locally-with-vagrant) tutorial you encounter OSIE at the section "Start the Worker".
+If you follow the [Vagrant Setup](/setup/locally-with-vagrant) tutorial you encounter OSIE at the section "Start the Worker".
```
Comment (geriadam) [inline docs/services/osie.md]: changed /setup/local-vagrant

File: docs/workflows.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ date: 2020-07-28
 
 # Workflows
 
-A workflow is the complete set of operations to be run on a Worker. It consists of two building blocks: a Worker's [hardware data](/about/hardware-data) and a [template](/about/templates). Workflows are immutable. Updating a template or hardware data does not update existing workflows.
+A workflow is the complete set of operations to be run on a Worker. It consists of two building blocks: a Worker's [hardware data](/hardware-data) and a [template](/about/templates). Workflows are immutable. Updating a template or hardware data does not update existing workflows.
```
Comment (geriadam) [inline docs/workflows.md]: changed to "/templates"

**Review by geriadam — APPROVED** [review]

File: docs/services/osie.md (hunk 1 above)
Comment (felixwidjaja) [reply docs/services/osie.md]: Thanks.

File: docs/workflows.md (hunk 2 above)
Comment (felixwidjaja) [reply docs/workflows.md]: Thanks.

**Pull request merged**